### PR TITLE
Contrib guide

### PR DESCRIPTION
**Pull request by grbsk**
Add the contributing guide.

We want to focus on the following:
- Where to get help, making sure people know Angular or Semantic UI issues should be addressed to those repos, not to us.
- How to submit issues, making a special point to ask people to search to see if the issue already exists first, and include as much information up front to help replicate and resolve an issue.
- How to request a feature or suggest one via a PR
- How to submit a PR

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Contributing to Semantic-UI-Angular
+
+We are grateful for the help and ideas of the AngularJs and Semantic-UI community. This guide will describe how _you_ can contribute and make this a first-rate module.
```
Comment (m0t0r) [inline CONTRIBUTING.md]: `AngularJs` => `AngularJS` :+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Once you have determined that you have an unreported problem, please use the following guidelines when opening an issue.
+
+- Overview of the Issue - Include any non-minified stack trace and other runtime errors that are available and relevant to the issue.
+- Use Case - explain why this is a bug for you
+- Angular, Semantic UI, and Semantic UI Angular Version(s) - Is this an issue with Angular, the Semantic UI CSS framework, or a regression issue with this module?
+- Browsers and Operating System - Is this a problem with all browsers, or specific browsers and/or specific OS? Also, be aware that browser requirements for Angular and Semantic UI apply and are the baseline requirements for this module.
+- Reproduce the Error - Please, please, please provide a live example (using Plunker or JSFiddle) or an unambiguous set of steps! Doing the footwork up front will allow us to replicate and fix the problem faster.
```
Comment (m0t0r) [inline CONTRIBUTING.md]: add links to Plunker and JSFiddle

Comment (m0t0r) [inline CONTRIBUTING.md]: Should we highlight  **provide a live example** ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **perf**: A code change that improves performance
+* **test**: Adding missing tests
+* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation generation
+
+### Scope
+
+The scope should be the aspect or subfeature being affected, such as a module, element, or collection. For example: `modules/checkbox`, 'elements/button', etc.
```
Comment (m0t0r) [inline CONTRIBUTING.md]: elements/button does not have proper formatting, it has quotes instead of backticks

**Comment by m0t0r** [comment]
@HackedByChinese LGTM so far, thanks. Are we going to have a separate guide for code style ?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+- **Major Change**: If you are interested in a major change, please [get in touch](#getting-help) with the team so we can coordinate our efforts, avoid duplicate work, and ensure your contributions are successfully accepted into the project.
+- **Minor Change**: If the change is small, you can fork the [repository][github] and create a Pull Request with your changes. See [Submitting a Pull Request](#pullrequest) for more information.
+
+## <a name="commiting"></a> Git Commit Guidelines
+
+We follow the `conventional-changelog` format used by the AngularJs team. You can follow the format manually, but we highly recommend you use a CLI wizard such as [Commitizen][commitizen].
+
+In fact, we have configured this repository to work with Commitizen. First, ensure Commitizen CLI is installed via `npm install -g commitizen`. Next, make sure the clone of this repository is installed via `npm install`. You're all set! Now, whenever you want to commit, just stage your changes e.g. `git add .` and commit using `git cz`. Follow the prompts and you're done.
+
```
Comment (m0t0r) [inline CONTRIBUTING.md]: Add a line about unit tests, for users before committing should run unit tests and fix linting warnings if there are any.

**Comment by m0t0r** [comment]
@HackedByChinese what do you think about [Github's support](https://help.github.com/articles/helping-people-contribute-to-your-project/) for Issues and PRs templates. I think it would be really nice to add it to the project.

**Comment by grbsk** [comment]
It makes sense to use it.

**Comment by matheuspoleza** [comment]
@m0t0r @HackedByChinese yes, I totally agree with that. I'm using in other project and It's really useful. It's easy to implement, are you gonna add in this PR?

**Comment by grbsk** [comment]
I'll add something and you guys can add your input to it.

**Comment by m0t0r** [comment]
Cool :+1:

**Comment by matheuspoleza** [comment]
:+1: @HackedByChinese

**Comment by matheuspoleza** [comment]
@HackedByChinese let's merge it? And what about code style? We can worry later about.

**Pull request closed without merging**